collaboration/1.slack.md

```diff
@@ -0,0 +1,101 @@
+# Slack
+
+[Slack](https://slack.com/) is a chat we use to discuss about anything.
```

Comment:
Just my 2cents, but I think knowing what Slack.com is in general is not what you're interested in while reading this. Slack.com is just a tool we decide to use to "implement" our team chat.